js/UcrBuildings.js

```diff
@@ -23,3 +23,8 @@ getBuildingLocation = function (bldg, room) {
 
     return "<a target=\"_blank\" href=\"" + url + "\">" + name + " " + room + "</a>";
 };
+
+getBuildingDesc = function(bldg) {
+  var name = buildingNames[bldg];
+  return name;
+}
```

Comment:
Nah, it doesn't really have to be. It's just force of habit of mine to have one file ask for a variable politely from another file rather than just go over and take it.